packages/editor-ui/src/constants.ts

```diff
@@ -34,7 +35,7 @@ export const BREAKPOINT_XL = 1920;
 
 
 // templates
-export const TEMPLATES_BASE_URL = `https://api.n8n.io/`;
+export const TEMPLATES_BASE_URL = `https://api-staging.n8n.io/`;
```

Comment:
I changed this just for testing purpose since the feature in the BE is working only on staging environment at the moment.

Reply:
reminder to change this